pygfx/geometries/_base.py

```diff
@@ -117,10 +117,23 @@ def bounding_box(self):
         if hasattr(self, "grid"):
             if self._aabb_rev == self.grid.rev:
                 return self._aabb
-            self._aabb = (
-                np.array([np.zeros_like(self.grid.data.shape), self.grid.data.shape])
-                - 0.5
-            )
+            grid_shape = list(self.grid.data.shape)
+            # if grid is multichannel, e.g. 3xf4
+            format = self.grid.format
+            if len(format) == 4 and format[1] == "x" and format[0] != "1":
+                # then strip last dimension from shape
+                grid_shape.pop(-1)
```

Comment:
I guess this could still go wrong if you supply e.g. a grayscale/single-channel 2D image but it still has a shape like `(320, 240, 1)`. Not sure how to account for that.

Reply:
Should we determine the number of dimensions somewhere, so you can use that to determine the appropriate shape downstream?

Reply:
I just noticed there is a `dim` property so I can use that

Reply:
> I just noticed there is a dim property so I can use that

Exactly